README.md

```diff
@@ -284,7 +284,7 @@ The following properties are available:
 * mscs-backup-location         - Location to store backup files.
 * mscs-backup-log              - Location of the backup log file.
 * mscs-backup-duration         - Length in days that backups survive.
-* mscs-log-duration            - Length in days that logs survive.
+* mscs-log-duration            - Length in days that logs survive. A value less than 1 disables log deletion.
```

Comment:
This change should be made to [Line 277 of msctl](https://github.com/MinecraftServerControl/mscs/blob/master/msctl#L277) as well